### ISSUE TITLE
'_SpecialEuclidean3Vectors' object has no attribute 'metric'

### ISSUE THREAD

**Issue by doublestrong** [geomstats/geomstats#1007]
hi, I am trying to make something like this:

`my_dim = 3`
`#we constrain the point type to be vector so the R3 and SE3 samples can be concatenated in .random_uniform`
`#cannot work for now as _SpecialEuclidean3Vectors' object has no attribute 'metric'`
`my_manifold = [SpecialEuclidean(n=my_dim, point_type='vector'), Euclidean(dim=my_dim)]`
`space_vector = ProductManifold(manifolds=my_manifold, default_point_type='vector')`

however, I encountered this error:
`AttributeError: '_SpecialEuclidean3Vectors' object has no attribute 'metric'`

I think same thing happens to SE2. So
`AttributeError: '_SpecialEuclidean2Vectors' object has no attribute 'metric'`

**Comment by nguigs** [comment]
Hi, thank you for raising this issue. This will be fixed in #1006.
In the mean time, you can use:
```python
from geomstats.geometry.special_euclidean import SpecialEuclidean, Euclidean
from geomstats.geometry.product_manifold import ProductManifold

dim = 3
se3 = SpecialEuclidean(n=dim, point_type='vector')
se3.metric = se3.left_canonical_metric
se3.metric.signature = (dim, 0)
my_manifold = [se3, Euclidean(dim)]
space_vector = ProductManifold(manifolds=my_manifold, default_point_type='vector')
```
Let me know if you run into other problems.
Also, beware that `SpecialEuclidean` is already the product of special orthogonal with R^3

**Comment by doublestrong** [comment]
Thanks for the answer. That's very helpful.
